domain-service/credit-application/src/main/resources/application.yml

```diff
@@ -0,0 +1,11 @@
+server:
+  port: 8080
+  error:
+    whitelabel:
+      enabled: false
+
+core-banking:
+  credit:
+    datasource:
+      mode: "mock"
```

Comment:
Should we maybe create a mock profile for this?

Reply:
Sure, but I'd rather skip it for the first iteration. We also have to add local dev profiles, I assume it will be easier to do them in bulk when we have all service skeletons and the docker compose.